core/keyserver/keyserver_test.go

```diff
@@ -56,9 +62,14 @@ func TestListEntryHistory(t *testing.T) {
 		// Test case setup.
 		c := &fakeCommitter{make(map[string]*tpb.Committed)}
 		st := &fakeSparseHist{make(map[int64][]byte)}
-		a := &fakeAppender{0, 0}
-		srv := New(c, fakeQueue{}, st, a, fakePrivateKey{}, fakeMutator{}, authentication.NewFake(), fakeFactory{})
-		if err := addProfiles(profileCount, c, st, a); err != nil {
+		fakeLog := fake.NewFakeTrillianClient()
+		admin := admin.NewStatic()
+		if err := admin.AddLog(logID, fakeLog); err != nil {
+			t.Fatalf("failed to add log to admin: %v", err)
+		}
+		sths := appender.NewTrillian(admin)
```

Comment:
Let's chat about this on GVC. 

This test is using a mock - `fakeLog`. Perhaps there's a better way to setup multi-tennant support - or delay multi tenant support till later.